### PR TITLE
Verazab style

### PR DESCRIPTION
**Pull request by nicholas-esterer**
Integrates a bunch of CSS that @VeraZab helped with, plus a bit of beautifying by adding icons for download and upload, have the size adapt to differently sized screens, and crediting of images.
App can be viewed at https://dash-playground.plotly.host/dash-image-proc-template/

### CODE REVIEW

**Comment by VeraZab** [comment]
nice, here's a color suggestion for links:
```
a {
    color: #a4f5f4;
}
```
Screen Shot 2020-06-11 at 4 19 21 PM

File: assets/style.css [hunk 1]
```diff
@@ -20,99 +20,102 @@ body, html {
...
+    #result-container, #sidebar {
+        flex-grow: initial;
+        width: 96%;
+    }
+    #image-controls-container {
+        width: 100%;
+    }
+    #image-container img {
+        width: 100%;
+        height: auto;
```
Comment (VeraZab) [inline assets/style.css:119]: i think its better without the height: auto, that way in mobile view the container does not grow at each rotation change

Comment (VeraZab) [reply assets/style.css:119]: also :
```
#image-container img {
    object-fit: contain;
}
```
has to be present on both mobile and desktop views if we don't want the image to distort when rotating

File: requirements.txt [hunk 2]
```diff
@@ -1,8 +1,43 @@
+appdirs==1.4.4
```
Comment (emmanuelle) [inline requirements.txt:1]: you will think I'm repeating myself :-) but a minimal `requirements.txt` obtained by cherry-picking the necessary packages (probably imports + pandas) will look better for a pedagogical template app. The number of deps looks a bit daunting at the moment.

**Pull request merged**